'make clean' broken

Issuing `make clean` results in:

```
make clean
fatal: not a git repository (or any of the parent directories): .git
make: *** [clean] Error 128
```

--- a/Makefile
+++ b/Makefile
@@ -50,7 +50,6 @@ clean:
 	@rm -rf version_number
 	@rm -rf src/*.o
 	@rm -rf src/*.so
-	@cd $(top_srcdir) && git reset --hard
 
 distclean: clean
 	@rm -rf $(PYVENV)

--- a/Makefile-cigraph
+++ b/Makefile-cigraph
@@ -206,6 +206,7 @@ pre_build: venv patches $(CSRC) $(CINC2) $(PARSER2) $(RSRC) $(RGEN) \
 
 clean:
 	rm -rf src/core src/vendor src/include src/Makevars.in src/Makevars.ucrt src/Makevars.win src/config.h.in src/rinterface.c R/aaa-auto.R
+	git -C $(top_srcdir) reset --hard
 
 .PHONY: all igraph force clean check check-cran check-rhub check-links install test
 

